homeassistant/components/ezviz/button.py

```diff
@@ -0,0 +1,130 @@
+"""Support for EZVIZ button controls."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from pyezviz.constants import SupportExt
+from pyezviz.exceptions import HTTPError, PyEzvizError
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DATA_COORDINATOR, DOMAIN
+from .coordinator import EzvizDataUpdateCoordinator
+from .entity import EzvizEntity
+
+PARALLEL_UPDATES = 1
+
+
+@dataclass
+class EzvizButtonEntityDescriptionMixin:
+    """Mixin values for EZVIZ button entities."""
+
+    method: Callable[[Any, Any, Any], Any]
```

Comment:
In the case of button, I understand the return value can be `Any`, but in the case of the provided parameters to this lambda, I think we can be more specific, right?

Reply:
Definitely, updated typing to be more specific.